docs/tables/steampipecloud_user_preferences.md

```diff
@@ -0,0 +1,20 @@
+# Table: steampipecloud_user_preferences
```

Comment:
Should the table be named `steampipecloud_user_preference`?

Reply:
@siddharthaturbot could you please add an example query output for my better understanding?